cookbook/case_studies/ml_training/house_price_prediction/README.rst

```diff
@@ -1,52 +1,65 @@
 House Price Regression
 -----------------------
 
-A house price prediction model is used to predict a house’s price, given the required inputs.
-In this example we'll train a model to predict the price of a house in multiple regions.
+House Price Regression refers to the prediction of house prices based on various factors, using the XGBoost Regression model (in our case).
+In this example, we will train our data on the XGBoost model to predict house prices in multiple regions.
 
-Generally, this could be accomplished using any regression model in machine learning.
+Generally, this can be accomplished using any regression model in Machine Learning.
+
+.. _flyte's-role:
 
 Where does Flyte fit in?
-========================
-- Orchestrates the machine learning pipeline
-- Can cache the output state between the steps (tasks as per Flyte)
-- Easier backtracking to the error source
-- Provides a Rich UI (if the Flyte backend is enabled) to view and manage the pipeline
+==========================
+- Orchestrates the machine learning pipeline.
+- Helps cache the output state between the steps (tasks as per Flyte).
+- Easier backtracking to the error source.
+- Provides a Rich UI (if the Flyte backend is enabled) to view and manage the pipeline.
 
 A typical house price prediction model isn’t dynamic, but a task has to be dynamic when multiple regions are involved. 
 
 To learn more about dynamic workflows, refer to `Write a dynamic task <https://docs.flyte.org/projects/cookbook/en/latest/auto/core/control_flow/dynamics.html#dynamic-workflows>`__.
 
 Dataset
-=======
-There is no built-in dataset that could be employed to build this model. A dataset has to be created, possibly using this reference model on `Github <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/multi_model_xgboost_home_value/xgboost_multi_model_endpoint_home_value.ipynb>`__.
+========
+There is no built-in dataset that could be employed to build this model. A dataset will have to be created, possibly using this reference model on `Github <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/multi_model_xgboost_home_value/xgboost_multi_model_endpoint_home_value.ipynb>`__.
 
 The dataset will have the following columns:
+
 - Price
+
 - House Size
+
 - Number of Bedrooms
+
 - Year Built
+
 - Number of Bathrooms
+
 - Number of Garage Spaces
+
 - Lot Size
 
 Steps to Build the Machine Learning Pipeline
-============================================
-- Generate dataset and split it into train, validation, and test datasets
-- Fit the XGBoost model on to the data
+==============================================
+- Generate the dataset, and split it into train, validation, and test subsets
+- Fit the XGBoost model to the data
 - Generate predictions
```

Comment:
I think we can remove this section as this is the typical ML flow. We need not specifically say this is what we are doing. What do you think?